drafts/WEP0001.rst

```diff
@@ -0,0 +1,282 @@
+
+Selection process
+-----------------
+
+All applications will be scored by the grant review team, which should
+consist of around four reviewers. The suggested tool is Grorg, but the
+grant review team may choose different tools.
+
+Reviewers can go through all applications one by one, and score them,
+possibly with a comment. Scores, applicant names, and comments from others
+should not be shown until the reviewer has provided a score. Scores can be
+updated, but these updates will be logged and visible.
+
+The scoring should be based on the overall impact that granting this person
+will have on them, the Write the Docs community, and the applicants wider
+community and country, compared to others.
+This is difficult to assess from a short form. Several guidelines are:
+
+* To what extent will letting this person attend have a further impact on the
+  community? For example, someone who essentially wrote “I will help my
+  company make more money” should likely be scored lower than someone who is
+  active in their local community.
+* The social impact of their work can be a part too. “I work to help this
+  bank I work for become richer” has less social impact than supporting someone
+  who said “I work on projects to help addicted homeless people towards recovery”,
+  and therefore the latter should be scored higher.
+* Where is this person from? Especially if they would take a large budget, but 
+  travel to a different Write the Docs conference would be much closer and/or cheaper,
+  that should be the preferred option. For example, an applicant from Germany
+  should generally apply for Prague instead of Portland.
+* Are they making a basic effort? For example, if someone doesn't even have an
+  estimate of the price of their flight without further explanation, they don't
+  appear very motivated.
+* Does this conference seem like an event that would benefit them? And do
+  they have some understanding of what the conference is about?
+
+After all scores are completed, the grant review team will discuss the top
+applications, prioritized by underrepresented/marginalized groups first,
+score second, and grant them unless discussion leads to another conclusion,
+until funds run out. The review team is not required to always follow the
+score based order. This also depends on the remaining resources.
+
+Speakers will bypass the review process and are immediately accepted, except
+in exceptional circumstances.
+
+Notifying accepted applicants
+-----------------------------
+
+* All accepted applicants will be notified by the grant review team.
+* Applicants need to confirm they are still intending to participate
+  within one week. This term can be extended if requested in the
+  application.
+* If they withdraw their application, the grant review team may be able
+  provide a few extra applicants with grants, prioritized by score.
+* Applicants that are very low on the list, and basically certain not to
```

Comment:
This seems tricky to get right.

Reply:
I'm not sure which part you think is tricky?

Reply:
I'm thinking about the time uncertainty around withdrawal and reallocation.

Reply:
The confirmation period is (except for special circumstances) one week, so I think we still have a good chance of accepting an additional few grants in the timeframe. In my experience, this is no more than a handful of cases. I think it makes sense to at least try.